README.md

```diff
@@ -4,33 +4,40 @@
+
+If you'd like to get Fabric through a package manager such as Bower, npm, or NuGet, check out the [package manager docs](https://github.com/OfficeDev/office-ui-fabric-core/blob/master/ghdocs/PACKAGES.md).
 
-Want to customize Fabric for your project? See [Building Fabric](https://github.com/OfficeDev/Office-UI-Fabric/blob/master/ghdocs/BUILDING.md) to learn about the build process.
+Want to customize Fabric for your project? See [building Fabric](https://github.com/OfficeDev/Office-UI-Fabric/blob/master/ghdocs/BUILDING.md) to learn about the build process.
 
 
 ##Learn more
 
-New to Fabric? Start with [What is Office UI Fabric?](http://dev.office.com/fabric/what-is-fabric) to understand its purpose, and then explore the full set of [styles](http://dev.office.com/fabric/styles) and [components](http://dev.office.com/fabric/components) to get a feel for what you can build. Components come in two flavors provided by the Fabric team: [JavaScript (Office UI Fabric JS)](https://github.com/OfficeDev/office-ui-fabric-js) and [React (Office UI Fabric React)](https://github.com/OfficeDev/office-ui-fabric-react).
+New to Fabric? Start with [the project's site](http://dev.office.com/fabric) to understand its purpose, and then explore the full set of [styles](http://dev.office.com/fabric/styles) to get a feel for what you can build. Components are contained in separate projects and come in two flavors provided by the Fabric team: [JavaScript (Office UI Fabric JS)](https://github.com/OfficeDev/office-ui-fabric-js) and [React (Office UI Fabric React)](https://github.com/OfficeDev/office-ui-fabric-react).
 
-When you're ready to create a simple app or Office Add-in with Fabric, follow along with the [Fabric Tutorial](https://github.com/OfficeDev/Office-UI-Fabric/blob/master/ghdocs/TUTORIAL.md) or [Use Office UI Fabric with Office Add-ins on MSDN](https://msdn.microsoft.com/EN-US/library/office/mt450443.aspx).
+When you're ready to create an Office Add-in with Fabric, take a look at the [Use Office UI Fabric with Office Add-ins on MSDN](https://msdn.microsoft.com/EN-US/library/office/mt450443.aspx) article.
 
 
 ##Contribute to Fabric
 
-Bug reports, feature requests, and questions are posted on the [issue tracker](https://github.com/OfficeDev/Office-UI-Fabric/issues). For details on our contribution process and how we label issues, see our [Contributing](https://github.com/OfficeDev/Office-UI-Fabric/blob/master/ghdocs/CONTRIBUTING.md) page.
+Bug reports, feature requests, and questions are posted on the [issue tracker](https://github.com/OfficeDev/Office-UI-Fabric-core/issues). For details on our contribution process and how we label issues, see our [contributing](https://github.com/OfficeDev/Office-UI-Fabric/blob/master/ghdocs/CONTRIBUTING.md) page.
```

Comment:
Gotcha. Nope, I don't feel strongly one way or another--just checking to see if it's intentional :)